frameworks/digital-outcomes-and-specialists-2/manifests/edit_service.yml

```diff
@@ -0,0 +1,125 @@
+-
+  name: Lab name
+  editable: False
+  questions:
+    - serviceName
+
+- name: Service essentials
+  editable: False
+  questions:
+    - helpGovernmentImproveServices
+    - bespokeSystemInformation
+    - dataProtocols
+    - openStandardsPrinciples
+
+- name: User research participants essentials
+  editable: False
+  questions:
+    - anonymousRecruitment
+    - manageIncentives
+
+- name: Team capabilities
+  editable: False
+  edit_questions: True
```

Comment:
What does the `edit_questions` key do? Feels like this should probably be false too? Or is it to do with summary table layout, I can't remember exactly.